book/theme/index.hbs

```diff
@@ -0,0 +1,384 @@
+<!DOCTYPE HTML>
+<html lang="{{ language }}" class="sidebar-visible no-js {{ default_theme }}">
+    <head>
+        <!-- Book generated using mdBook -->
+        <meta charset="UTF-8">
+        <title>{{ title }}</title>
+        {{#if is_print }}
+        <meta name="robots" content="noindex" />
+        {{/if}}
+        {{#if base_url}}
+        <base href="{{ base_url }}">
+        {{/if}}
+
+        <script async src="https://www.gstatic.com/brandstudio/kato/cookie_choice_component/cookie_consent_bar.v3.js"
+                data-autoload-cookie-consent-bar="true"></script>
+
+        <script async src="https://www.googletagmanager.com/gtag/js?id=G-ZN78TEJMRW"></script>
+        <script>
+          window.dataLayer = window.dataLayer || [];
+          function gtag(){dataLayer.push(arguments);}
+          gtag('js', new Date());
+          gtag('config', 'G-ZN78TEJMRW');
+        </script>
```

Comment:
Hey we do NOT track users! Seems that you copied the file but forgot to delete? ;)

Reply:
Yeah, please don't use the Google Analytics from my book — that'll be confusing :smile:

I think it would be easier to start with https://github.com/rust-lang/mdBook/blob/master/src/theme/index.hbs and then take the parts of https://github.com/google/comprehensive-rust/commits/main/theme/index.hbs that you need/want.

Reply:
deleted